content/views.py

```diff
@@ -62,14 +74,44 @@ def articles(self, request, pk=None):
         return paginator.get_paginated_response(serialized)
 
 
+class PublishedIssueViewSet(ReadOnlyModelViewSet):
+    queryset = Issue.objects.filter(publish_date__isnull=False)
+    serializer_class = IssueSerializer
+    filter_backends = [SearchableFilterBackend]
+    search_fields = []
+    search_transformers = {"__term": transform_issue_name}
+
+    __articles_filter = SearchableFilterBackend(["title", "content_raw"])
+
+    @action(detail=False, methods=["GET"])
+    def latest(self, request):
+        latest = self.get_queryset().first()
+        return Response(IssueSerializer(latest).data)
+
+
 class ArticleViewSet(ModelViewSet):
+    class ArticlePermissions(IsPublishedOrIsAuthenticated):
```

Comment:
I believe this means we're no longer using `IsArticleOwnerOrReadOnly`, so it should be deleted.

Reply:
I'm not very comfortable with deleting `IsArticleOwnerOrReadOnly` even if it isn't used, since we might find use later on. Or do you mean removing the imports?

Reply:
Well, I did mean deleting the permission, though I take your point on future use. Removing the import is a good idea for hygiene reasons though.

Reply:
Import was removed in 1bba966.